src/connection/tests.rs

```diff
@@ -822,7 +822,7 @@ fn from_connect_fetchall_rollback_panic_closed() {
 
 #[test]
 #[serial]
-#[should_panic(expected = "Can't commit while executing")]
```

Comment:
This was a "double" copy-paste bug (first in the code and then test just followed) :smile: